api/all.yaml

```diff
@@ -1414,6 +1702,39 @@ components:
           description: |
             Opaque token to retrieve the next page of results. Absent if there are no more pages.
             __page_token__ should be set to this value for the next request (for the next page of results).
+    GenerateTemporaryModelCredential:
+      example:
+        table_id: model_id
+        operation: null
+      properties:
+        table_id:
+          description: |
```

Comment:
Would probably be helpful to have more details about when to generate these credentials, especially in the context of OSS